.aws/src/event-rules/user-registration/eventConfig.ts

```diff
@@ -0,0 +1,8 @@
+import { config as globalConfig } from '../../config';
+
+export const eventConfig = {
+  name: 'UserRegistration',
+  source: 'web-repo',
+  detailType: ['User Registration'],
```

Comment:
should match to: https://github.com/Pocket/Web/blob/179947cbae91f4451094906a34b589efcdcc09dd/tests/functional/AWS/EventBridge/SubscriberTest.php#L98